bbot/test/test_step_2/module_tests/test_module_trufflehog.py

```diff
@@ -868,7 +868,7 @@ def check(self, module_test, events):
             [
                 e
                 for e in filesystem_events
-                if e.data["path"].endswith("/git_repos/test_keys") and Path(e.data["path"]).is_dir()
+                if e.data["path"].endswith("/git_repos/.bbot_test/test_keys") and Path(e.data["path"]).is_dir()
```

Comment:
should this be `/git_repos/blacklanternsecurity/test_keys`?

Reply:
The filter patches it to clone from a local repository instead
```python
temp_path = Path("/tmp/.bbot_test")

def new_filter_event(event):
            event.data["url"] = event.data["url"].replace(
                "https://github.com/blacklanternsecurity", f"file://{temp_path}"
            )
            return old_filter_event(event)
```